docs/podman-network-create.1.md

```diff
@@ -0,0 +1,70 @@
+% podman-network-create(1)
+
+## NAME
+podman\-network-create- Create a Podman CNI network
+
+## SYNOPSIS
+**podman network create**  [*options*] name
+
+## DESCRIPTION
+Create a CNI configuration for use with Podman. At the time of this writing, the only network
```

Comment:
Maybe `CNI-network configuration`? It's a bit surprising to change the terminology right at the beginning.